src/l500/l500-options.h

```diff
@@ -25,7 +25,35 @@ namespace librealsense
         get_current = 0,
         get_min = 1,
         get_max = 2,
-        get_step = 3
+        get_step = 3,
+        get_default = 4
+    };
+
+    class l500_options;
+
+    class digital_gain_option : public uvc_xu_option< int >
+    {
+    public:
+        digital_gain_option( uvc_sensor & ep,
+                             platform::extension_unit xu,
+                             uint8_t id,
+                             std::string description,
+                             const std::map< float, std::string > & description_per_value,
+                             firmware_version fw_version,
+                             l500_options * owner )
+            : uvc_xu_option< int >( ep, xu, id, description, description_per_value )
```

Comment:
When I have class hierarchies I like to do this as the first line in a class:
```
    typedef uvc_xu_option< int > super;
```
And then I can just:
```
            : super( ep, xu, id, description, description_per_value )
```
Or
```
        uvc_xu_option< int >::set( value );
```
Up to you. I think it's easier to write and read without having to remember which is the base class, and if the base class changes then it's only in one place.

Reply:
took your tip :)